models/Comment.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const CommentSchema = new mongoose.Schema({
+        
+    comment:{
+        type: String,
+        required: true
+    },
+   
+    tweetId: {
```

Comment:
I believe you should have `required:true` here

Reply:
Yes sir , i have